example/ios/Runner/Info.plist

```diff
@@ -41,5 +41,9 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSAppTransportSecurity</key>
+     <dict>
+          <key>NSAllowsArbitraryLoads</key><true/>
+     </dict>
```

Comment:
just flagging that according to the docs this configuration flag will need to be justified to Apple to get this code in the app store. i'm not familiar enough with this project to know if that is a concern or if this is just some debug code to allow non-https protocol requests. just noting it may come up

>You must supply a justification during App Store review if you set the key’s value to YES, as described in Provide Justification for Exceptions. Use this key with caution because it significantly reduces the security of your app. In most cases, it’s better to upgrade your servers to meet the requirements imposed by ATS, or at least to use a narrower exception.

https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads

Reply:
updated to add this context for future readers